content/en/docs/instrumentation/js/serverless.md

```diff
@@ -0,0 +1,166 @@
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const {
+  getNodeAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-node");
+
+api.diag.setLogger(new api.DiagConsoleLogger(), api.DiagLogLevel.ALL);
+
+const provider = new NodeTracerProvider();
+const collectorOptions = {
+  url: "<backend_url>",
+};
+
+const spanProcessor = new BatchSpanProcessor(
+  new OTLPTraceExporter(collectorOptions)
+);
+
+provider.addSpanProcessor(spanProcessor);
+provider.register();
+
+registerInstrumentations({
+  instrumentations: [
+    getNodeAutoInstrumentations({
+      "@opentelemetry/instrumentation-aws-lambda": {
+        disableAwsContextPropagation: true,
+      },
+    }),
+  ],
+});
+```
+
+Replace `<backend_url>` with the URL of your favorite backend to export all
+traces to it. If you don't have one setup already, you can check out [Jaeger](https://www.jaegertracing.io/) or [Zipkin](https://zipkin.io/).
+
+Note that `disableAwsContextPropagation` is set to true. The reason for this is
+that the Lambda instrumentation tries to use the X-Ray context headers by
+default, this results in a non-sampled context, which creates a
+`NonRecordingSpan`.
+
+More details can be found in the instrumentation
+[documentation](https://www.npmjs.com/package/@opentelemetry/instrumentation-aws-lambda).
+
+### AWS Lambda function handler
+
+Now that you have a Lambda wrapper, you can create a simple handler that will
+serve as a Lambda function. Save the following code as `handler.js`.
+
+```javascript
+/* handler.js */
+
+"use strict";
+
+const https = require('https');
+
+function getRequest() {
+  const url = 'https://opentelemetry.io/';
+
+  return new Promise((resolve, reject) => {
+    const req = https.get(url, res => {
+      resolve(res.statusCode);
+    });
+
+    req.on('error', err => {
+      reject(new Error(err));
+    });
+  });
+}
+
+exports.handler = async (event) => {
+  try {
+    const result = await getRequest();
+    return {
+      statusCode: result,
+    };
+  } catch (error) {
+    return {
+      statusCode: 400,
+      body: error.message,
+    };
+  }
+};
+```
+
+### Deployment
+
+There are multiple ways of deploying, but we will be using [Serverless Framework](https://github.com/serverless/serverless)
+for ease of use. Create a file called `serverless.yml`:
+
+```yaml
+service: lambda-otel-native
+frameworkVersion: "3"
+provider:
+  name: aws
+  runtime: nodejs14.x
+  region: "<your-region>"
+  environment:
+    NODE_OPTIONS: --require lambda-wrapper
+functions:
+  lambda-otel-test:
+    handler: handler.hello
+```
+
+For OpenTelemetry to work properly, `lambda-wrapper.js` must be included before
+any other file: the `NODE_OPTIONS` setting ensures this.
+
+Note if you are not using Serverless Framework to deploy your Lambda function,
+you must manually add this environment variable using the AWS Console UI.
```

Comment:
This seems to me like we need an additional section for how to properly package and deploy the app for Lambda when not using serverles.